cmd/broker/update.go

```diff
@@ -31,14 +31,19 @@ func NewUpdateProcessingQueue(ctx context.Context, manager *process.StagedManage
 		},
 		{
 			stage:     "cluster",
-			step:      update.NewUpgradeShootStep(db.Operations(), db.RuntimeStates(), provisionerClient),
+			step:      update.NewUpgradeShootStep(db.Operations(), db.RuntimeStates(), provisionerClient, cli),
```

Comment:
The step need `cli` to read Runtime resource and decide if should be executed or skipped